jenkins/pipelines/cd/pre-release-community-docker-rocky.groovy

```diff
@@ -79,7 +16,10 @@ def get_dockerfile_url(product, is_enterprise, is_debug){
 }
 
 def get_image_str_for_community(product, arch, tag, is_failpoint, is_debug) {
-    def imageTag = tag + "-rocky"+ "-pre"
+    def imageTag = params.IMAGE_TAG
+    if (! imageTag){
+        imageTag = tag + "-rocky"+ "-pre"
+    }
```

Comment:
why use global variable in this function? can we pass `tag` argument to call this function?

Reply:
fixed